src/azure-cli/azure/cli/command_modules/acs/_consts.py

```diff
@@ -49,3 +49,6 @@
 }
 
 CONST_CANIPULL_IMAGE = "mcr.microsoft.com/aks/canipull:0.0.2-alpha"
+
+CONST_MANAGED_IDENTITY_OPERATOR_ROLE = 'Managed Identity Operator'
+CONST_MANAGED_IDENTITY_OPERATOR_ROLE_ID = 'f1a07417-d97a-45cb-824c-7a7467783830'
```

Comment:
Why do we need to hard code a ROLE_ID ?

Reply:
That's built-in role id similar as contributor: https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles , it is used to detect whether the role assignment already exists

Reply:
Got it, thanks~